projects/kyber/index.js

```diff
@@ -100,26 +101,50 @@ query get_tvl($block: Int) {
       }
     });
     const {dmmFactory} = await request(
-      graphUrl,
-      graphQuery,
+      egraphUrl,
+      egraphQuery,
       {
         block,
       }
+      
     );
     if(dmmFactory !== null){ // Has been created
-      const dmmTvlInUsdt = (Number(dmmFactory.totalLiquidityUSD)* 1e6).toFixed(0)
-      sdk.util.sumSingleBalance(balances, usdtAddress, dmmTvlInUsdt)
+      const ethTVL = (Number(dmmFactory.totalLiquidityUSD)* 1e6).toFixed(0)
+      sdk.util.sumSingleBalance(balances, usdtAddress, ethTVL)
     }
 
     return balances;
   }
+  async function polygonTvl(){
+    var polygonEndpoint ='https://api.thegraph.com/subgraphs/name/piavgh/dmm-exchange-matic';
+      var query = gql`
+      {
+        dmmFactories(first: 1) {
+          totalLiquidityUSD,
+        }
+      }
+      `;
+    var graphQLClient = new GraphQLClient(polygonEndpoint)
+    const results = await retry(async bail => await graphQLClient.request(query))
+    return {
+      [usdtAddress]: BigNumber(results.dmmFactories[0].totalLiquidityUSD)
+      .multipliedBy(10 ** 6)
+      .toFixed(0),
+    }
 
+  }
 /*==================================================
   Exports
 ==================================================*/
 
   module.exports = {
-    ethereum: tvl,
-    start: 1546515458,  // Jan-03-2019 11:37:38 AM +UTC
-    tvl,
+
+    ethereum: {
+      tvl: ethTvl,
+    start: 1617606651, // @5-Apr-2021, 7:10:51 AM+UTC 
```

Comment:
`start` should be outside the ethereum object, although you could also remove it

Reply:
ok